examples/tutorials/Working_With_Splitters.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        tasks, datasets, transformers = dc.molnet.load_bace_classification(featurizer='ECFP', splitter=splitter)
</pre><p>any specific reason we are switching from tox21 dataset to BACE?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3837/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>